format/FormatHDF5EigerNearlyNexus.py

```diff
@@ -101,7 +100,7 @@ def create_scalar(handle, path, dtype, value):
 
         # Add detector type
         create_scalar(
-            handle["entry/instrument/detector"], "type", "S4", numpy.string_("PIXEL")
+            handle["entry/instrument/detector"], "type", "S4", np.string_("PIXEL")
```

Comment:
S4 is length 4 string, PIXEL has 5 letters? 🤔

Reply:
probably copy-pasta from a few lines above where it is S4 "NXmx".

Reply:
1538c069c75046dbb21b98a50e86bae887d56727 resolved